src/sentry/api/helpers/group_index.py

```diff
@@ -929,18 +929,39 @@ def update_groups(request, group_ids, projects, organization_id, search_fn):
 
     if "assignedTo" in result:
         assigned_actor = result["assignedTo"]
+        assigned_by = (
+            request.data.get("assignedBy")
+            if request.data.get("assignedBy") in ["assignee_selector", "suggested_assignee"]
+            else None
+        )
         if assigned_actor:
             for group in group_list:
                 resolved_actor = assigned_actor.resolve()
 
-                GroupAssignee.objects.assign(group, resolved_actor, acting_user)
+                created = GroupAssignee.objects.assign(group, resolved_actor, acting_user)
+                analytics.record(
+                    "manual.issue_assignment",
+                    organization_id=project_lookup[group.project_id].organization_id,
+                    project_id=group.project_id,
+                    group_id=group.id,
+                    assigned_by=assigned_by,
+                    had_to_deassign=(not created),
```

Comment:
how would you feel about making this `reassigned` (still either `True` or `False`) and then a separate event `ManualIssueUnassignment` ?

Reply:
@MeredithAnya ehh wouldn't that be a  redundant event? Since we know from the boolean.

Reply:
@NisanthanNanthakumar I was thinking that reassignment is when it goes from being assigned to being assigned to someone else. And then unassignment (or deassignment) as going from being assigned to not assigned at all.

Reply:
@MeredithAnya hmm well you have to `deassign` to `reassign`. And I think the value is in tracking `deassignment` bc that tells us that Code Owners/Issue Owners were wrong

Reply:
@NisanthanNanthakumar okays, I think whatever makes the most sense when doing queries later. I would change `had_to_deassign` to just be `deassigned` though